fuzzbucket_client/__main__.py

```diff
@@ -1215,17 +1247,19 @@ def _build_request(self, url, data=None, headers=None, method="GET"):
         req.headers["Fuzzbucket-Secret"] = self._secret
         return req
 
-    def _resolve_sshable_box(self, box):
-        matching_box = self._find_box(box)
+    def _resolve_sshable_box(self, box_name: str) -> tuple[typing.Optional[Box], bool]:
+        matching_box = self._find_box(box_name)
         if matching_box is None:
-            log.error(f"no box found matching {box!r}")
+            log.error(f"no box found matching {box_name!r}")
             return None, False
         if matching_box.get("public_dns_name") is None:
             log.error(f"no public dns name found for box={matching_box['name']}")
             return None, False
         return matching_box, True
 
-    def _build_ssh_command(self, box, unknown_args):
+    def _build_ssh_command(
+        self, box: Box, ssm: bool, unknown_args: typing.List[str]
```

Comment:
```suggestion
        self, box: Box, ssm: bool, unknown_args: list[str]
```

Reply:
Funny you should mention! In this particular case when I used `list[str]`, pyright was *very upset* for reasons I couldn't untangle 😬 

![image](https://github.com/rstudio/fuzzbucket/assets/45143/ae2ade18-7e6a-401c-b773-27edc99cf6a9)

Reply:
Ope, found it! Silly of me to use the name `list` as a method 😬